packages/babel-types/src/index.js

```diff
@@ -1,601 +1,126 @@
+} from "./comments/inheritInnerComments";
+export {
+  default as inheritLeadingComments,
+} from "./comments/inheritLeadingComments";
+export { default as inheritsComments } from "./comments/inheritsComments";
+export {
+  default as inheritTrailingComments,
+} from "./comments/inheritTrailingComments";
+export { default as removeComments } from "./comments/removeComments";
 
-  for (const key in node) {
-    if (key[0] === "_" && node[key] != null) node[key] = undefined;
-  }
+// constants
+export * from "./constants/generated";
+export * from "./constants";
 
-  const syms: Array<Symbol> = Object.getOwnPropertySymbols(node);
-  for (const sym of syms) {
-    node[sym] = null;
-  }
-}
+// converters
+export { default as ensureBlock } from "./converters/ensureBlock";
+export {
+  default as toBindingIdentifierName,
+} from "./converters/toBindingIdentifierName";
+export { default as toBlock } from "./converters/toBlock";
+export { default as toComputedKey } from "./converters/toComputedKey";
+export { default as toExpression } from "./converters/toExpression";
+export { default as toIdentifier } from "./converters/toIdentifier";
+export { default as toKeyAlias } from "./converters/toKeyAlias";
+export {
+  default as toSequenceExpression,
+} from "./converters/toSequenceExpression";
+export { default as toStatement } from "./converters/toStatement";
+export { default as valueToNode } from "./converters/valueToNode";
 
-export function removePropertiesDeep(tree: Node, opts?: Object): Node {
-  traverseFast(tree, removeProperties, opts);
-  return tree;
-}
+// definitions
+export * from "./definitions";
 
-//
+// modifications
 export {
-  getBindingIdentifiers,
-  getOuterBindingIdentifiers,
-} from "./retrievers";
-
+  default as appendToMemberExpression,
+} from "./modifications/appendToMemberExpression";
+export { default as inherits } from "./modifications/inherits";
 export {
-  isBinding,
-  isReferenced,
-  isValidIdentifier,
-  isValidES3Identifier,
-  isLet,
-  isBlockScoped,
-  isVar,
-  isSpecifierDefault,
-  isScope,
-  isImmutable,
-  isNodesEquivalent,
-} from "./validators";
-
+  default as prependToMemberExpression,
+} from "./modifications/prependToMemberExpression";
+export { default as removeProperties } from "./modifications/removeProperties";
 export {
-  toComputedKey,
-  toSequenceExpression,
-  toKeyAlias,
-  toIdentifier,
-  toBindingIdentifierName,
-  toStatement,
-  toExpression,
-  toBlock,
-  valueToNode,
-} from "./converters";
+  default as removePropertiesDeep,
+} from "./modifications/removePropertiesDeep";
+export {
+  default as removeTypeDuplicates,
+} from "./modifications/flow/removeTypeDuplicates";
 
+// retrievers
+export {
+  default as getBindingIdentifiers,
+} from "./retrievers/getBindingIdentifiers";
+export {
+  default as getOuterBindingIdentifiers,
+} from "./retrievers/getOuterBindingIdentifiers";
+
+// traverse
+export { default as traverse } from "./traverse/traverse";
```

Comment:
Would it be better to include the types in this file and do `export type * from "./traverse/traverse";` here?

Reply:
sure, will do that.